.travis.yml

```diff
@@ -8,23 +8,27 @@ git:
 branches:
   only:
     - develop
-    - "/^epic\\/.*$/"
-    - "/^hotfix\\/.*$/"
     - "/^release\\/.*$/"
-env:
-  - TEST_SUITE="./build.sh sonar"
-  - TEST_SUITE="./ci-scripts/e2e-cypress.sh"
-script: $TEST_SUITE
-deploy:
-  provider: script
-  skip_cleanup: true
-  script: bash scripts/publish-builds.sh
-  on:
-    branch: develop
-notifications:
-  email:
-    on_failure: always
-    on_success: change
+defaults: &defaults
+  script: "./ci-scripts/e2e-cypress.sh -s regression"
+jobs:
+  - script: "./build.sh sonar"
+  - script:
```

Comment:
Can we add mandatory job for smoke tests and keep the regression parallel as optional for now until we identify and fix flakiness in few tests?

Reply:
What would make it optional?

Reply:
Allow this jobs to fail and still return green status if smole/build jobs pass

Reply:
I think we can do that. But the main reason to do this is to avoid people to merge unless all the e2e tests pass. What would be the advantage of doing this without failing the build?

Reply:
It would be temporary just to identify and fix all flaky in parallel tests. Once that is done we switch it to normal required checks.

Reply:
Something for 2-3 weeks to see the flakiness on bigger data set that few runs in this PR.

Reply:
I tested this today and it seems that [the option to allow failures](https://docs.travis-ci.com/user/conditional-builds-stages-jobs/#conditionally-allowing-jobs-to-fail) it is not supported with parallelization unfortunately. We can wait until we stabilize the e2es or we can do some runs to gather statistics. What do you think?

Reply:
Ok. I can tackle stabilization of e2e tests for parallelization as a first master ticket in maintenance/quality initiative.